### PR TITLE
Add a Resources page to the website

### PR DESCRIPTION
**Pull request by rcgenova**

### CODE REVIEW

File: website/source/resources.html.erb [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+layout: "inner"
+page_title: "Resources"
+description: |-
+  Nomad is widely deployed across a range of enterprises and business verticals..
```
Comment (dadgar) [inline website/source/resources.html.erb]: Two periods at end

File: website/source/resources.html.erb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  <li><strong><a href="https://aws.amazon.com/quickstart/architecture/nomad/">HashiCorp Nomad on AWS</a> - AWS Quickstart Guide for Nomad</strong></li>
+  <li><strong><a href="https://github.com/hashicorp/nomad/tree/master/terraform">Nomad Sandbox Environment for AWS and Azure</a> - Easily Provision a Combined Nomad, Consul and Vault Cluster on AWS or Azure</strong></li>
+  <li><strong><a href="https://www.katacoda.com/hashicorp/scenarios/nomad-introduction">Introduction to Nomad (Katacoda)</a> - Interactive Introduction to Nomad</strong></li>
+  <li><strong><a href="https://katacoda.com/hashicorp/scenarios/playground">Nomad Playground (Katacoda)</a> - Online Nomad Playground Environment</strong></li>
+</ul>
+
+<h2>Integrations</h2>
+
+<ul>
+  <li><strong><a href="https://verizon.github.io/nelson/">Verizon Nelson</a> - Automated, Multi-region Container Deployment with HashiCorp Nomad</strong></li>
```
Comment (dadgar) [inline website/source/resources.html.erb]: DataDog would be a good one to add

Comment (rcgenova) [reply website/source/resources.html.erb]: It doesn't appear to be live on their site yet.

File: website/source/resources.html.erb (hunk 1 above)
Comment (rcgenova) [reply website/source/resources.html.erb]: Fixed.

File: website/source/resources.html.erb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  <li><strong>Citadel</strong></li>
+  <li><strong>Deluxe Entertainment</strong></li>
+  <li><strong>Elsevier</strong></li>
+  <li><strong>Jet.com</strong></li>
+  <li><strong>Pagerduty</strong></li>
+  <li><strong>SAP Ariba</strong></li>
+  <li><strong>Spaceflight Industries</strong></li>
+  <li><strong>SpotInst</strong></li>
+  <li><strong>UnderArmour</strong></li>
+  <li><strong>Verizon</strong></li>
```
Comment (jippi) [inline website/source/resources.html.erb]: can add Bownty.com and SeatGeek as well :)

Comment (dadgar) [reply website/source/resources.html.erb]: Send a PR our way!

Comment (rcgenova) [reply website/source/resources.html.erb]: Added!

**Pull request merged**